src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/additions/IndividualManagingOfficerAddition.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.additions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.Address;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.PersonName;
+
+import java.time.LocalDate;
+
+public class IndividualManagingOfficerAddition extends ManagingOfficerAddition{
```

Comment:
According to the mapping documentation, we need to set change=officerAppointmentChange for both individual and corporate MOs